lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/grok/Match.java

```diff
@@ -236,6 +231,7 @@ else if ((tmp[0] == '"' && tmp[value.length() - 1] == '"')
      * @return Json of the matched element in the text
      */
     public String toJson(Boolean pretty)
+            throws JsonProcessingException
```

Comment:
Instead of exporting a checked `JsonProcessingException` into all caller signatures, let's just catch and handle that exception locally within this function and re-throw a `RuntimeException`